pkg/environment/README.md

```diff
@@ -0,0 +1,35 @@
+# Environment module
+
+This module is used to get information about the running environment of a node.
```

Comment:
is that os.Getenv or that's the kernel passed arguments or what specifically?

Reply:
It defines different constants based on the current running environment, but it uses both to get them.